hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,5 @@
+RELEASE_TYPE: minor
+
+This release disallows using :obj:`python:typing.TypeAlias`
+with :func:`~hypothesis.strategies.from_type`
+and :func:`~hypothesis.strategies.register_type_strategy`.
```

Comment:
Let's also say *why* here, and reference `:issue:2978` for details so that users can chime in if it affects them.